sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -47,364 +48,444 @@ const alwaysBeDeletedSubscription = "alwaysbedeletedsubscription";
+      should.equal(
+        response.message.startsWith("The messaging entity") ||
+          response.message.startsWith("Entity") ||
+          response.message.startsWith("SubCode") ||
+          response.message.startsWith("No service"),
+        true,
+        `Expected error message to be a textual content but got "${response.message}"`
+      );
+    });
+
+    it(`Gets an existent ${testCase.entityType} entity successfully`, async () => {
+      let response;
+
+      response = await getEntity(
+        testCase.entityType,
+        testCase.alwaysBeExistingEntity,
+        testCase.parentTopicName,
+        testCase.parentSubscriptionName
+      );
+      should.equal(
+        response[testCase.entityType.toLowerCase() + "Name"],
+        testCase.alwaysBeExistingEntity,
+        "Entity name mismatch"
+      );
+    });
+
+    it(`Deletes a non-existent ${testCase.entityType} entity returns an error`, async () => {
+      let response;
+
+      try {
+        await deleteEntity(
+          testCase.entityType,
+          "notexisting",
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        response = err;
+      }
+
+      should.equal(response.statusCode, 404);
+      should.equal(
+        response.message.startsWith("The messaging entity") ||
+          response.message.startsWith("Entity") ||
+          response.message.startsWith("SubCode") ||
+          response.message.startsWith("No service"),
+        true,
+        `Expected error message to be a textual content but got "${response.message}"`
+      );
+    });
+
+    it(`Deletes an existent ${testCase.entityType} entity successfully`, async () => {
+      let response;
+
+      try {
+        await createEntity(
+          testCase.entityType,
+          testCase.alwaysBeDeletedEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        log.httpAtomXml("Ignoring clean up step");
+      }
+      response = await deleteEntity(
+        testCase.entityType,
+        testCase.alwaysBeDeletedEntity,
+        testCase.parentTopicName,
+        testCase.parentSubscriptionName
+      );
+
+      should.equal(response._response.status, 200);
+    });
+
+    it(`Get on non-existent ${testCase.entityType} entity throws an error`, async () => {
+      let response;
+      try {
+        await deleteEntity(
+          testCase.entityType,
+          testCase.alwaysBeExistingEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        log.httpAtomXml("Ignoring clean up step");
+      }
+      try {
+        response = await getEntity(
+          testCase.entityType,
+          testCase.alwaysBeExistingEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        response = err;
+      }
+      try {
```

Comment:
Why do we create an entity here?

Reply:
Mocha doesn't allow for specific before/after to be added, thus ones that are test specific are included within the test